reporting/src/request/index.js

```diff
@@ -180,7 +180,17 @@ export default class RequestMonitor {
     });
 
     // load the whitelist async - qs protection will start once it is ready
-    this.qs_whitelist.init();
+    (async () => {
```

Comment:
I was actually thinking that it is better to just crash the whole startup. Cannot we await instead?